### ISSUE TITLE
Super long genotype labels

### ISSUE THREAD

**Issue by yuanzhou** [monarch-initiative/monarch-legacy#1025]
Some genotypes have super long labels in a format that I'll need to look up Guidelines for Nomenclature of Mouse and Rat Strains (http://www.informatics.jax.org/mgihome/nomen/strains.shtml#mice) but still have hard times understanding. 

![capture](https://cloud.githubusercontent.com/assets/195873/10614943/fafedd18-772a-11e5-85f0-4aeb1d20c8aa.PNG)

So I'm adding this ticket to create some discussions on this issue to see if there's a better way to render the genotype long labels.

**Comment by kshefchek** [comment]
cc @nlwashington and @mbrush, I believe these are the correct labels, could we shorten labels over a certain length and add an ellipsis at the end?  This has been my approach for the barcharts.

**Comment by yuanzhou** [comment]
@kshefchek I agree with you in terms of shortening the labels and add ellipsis. That's what we did for the long labels on the x and y axis. I'm also questioning myself if the full-length of the long label makes any difference from the half-length for example. We still want to click to see the genotype.

**Comment by jmcmurry** [comment]
@yuanzhou I've not seen this show up in a long time. Is it still an issue?
